spec/features/cards_managment_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe "Card management", :type => :feature do
+    
+  let!(:card) { create :card }
+  
+  it "User creates a new card" do
+    visit new_card_path
+
+    fill_in 'card_original_text', with: "My Card"
+    fill_in 'card_translated_text', with: "Моя карта"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.